Build/Chakra.Build.props

```diff
@@ -25,14 +25,7 @@
     <!-- ARM does not support ICU until we figure out how to link icudtXXl.dat without genccode.exe -->
     <ChakraICU Condition="'$(Platform)'=='ARM'">false</ChakraICU>
```

Comment:
Nit: considering this variable is not truly a Boolean, I'd use another value here. disabled?

Reply:
In this interest of getting this through, I can make an issue for this and work on it later? False is used even in current master for turning this setting off, though I agree that disabled is nicer.

Reply:
Fine by me